behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/personopplysning/PersonInformasjonBuilder.java

```diff
@@ -76,6 +77,10 @@ public PersonInformasjonBuilder leggTil(PersonopplysningBuilder builder) {
         return this;
     }
 
+    public PersonInformasjonBuilder leggTil(Function<PersonInformasjonBuilder, PersonopplysningBuilder> function) {
```

Comment:
Hva er tanken med denne ?

Reply:
Tok det vekk da det er litt utenfor scope, men la det til for å gjøre `leggTil` mer funksjonell. Da kan man bruke den slik:

```diff
-        var builder = PersonInformasjonBuilder.oppdater(Optional.empty(), PersonopplysningVersjonType.REGISTRERT);
-        builder.leggTil(
-                 builder1.getPersonopplysningBuilder(aktørId).medFødselsdato(fødselsdato));

+         PersonInformasjonBuilder.oppdater(Optional.empty(), PersonopplysningVersjonType.REGISTRERT)
+                .leggTil(builder -> 
+                        builder.getPersonopplysningBuilder(aktørId).medFødselsdato(fødselsdato));
```

Reply:
Vi har noe å gå opp i alle builderpatterns. Spesielt at vi sender buildere rundt for å legge til ting. Men det er litt egen aktivitet.